Makefile

```diff
@@ -47,7 +47,8 @@ E2E_DATA_DIR ?= $(REPO_ROOT)/test/e2e_new/data
 E2E_CONF_PATH  ?= $(E2E_DATA_DIR)/e2e_conf.yaml
 KUBETEST_CONF_PATH ?= $(abspath $(E2E_DATA_DIR)/kubetest/conformance.yaml)
 KUBETEST_FAST_CONF_PATH ?= $(abspath $(REPO_ROOT)/test/e2e_new/data/kubetest/conformance-fast.yaml)
-CONFORMANCE_CI_TEMPLATE := $(ARTIFACTS)/templates/cluster-template-conformance-ci-artifacts.yaml
+CONFORMANCE_CI_TEMPLATE := _artifacts/templates/cluster-template-conformance-ci-artifacts.yaml
```

Comment:
Is this related to something hardcoded in a kustomize file? I'm wondering if using envsubst or similar might be better than having a hardcoded path to _artifacts

Reply:
Maybe something we could add to the e2e config parser?

Reply:
Let me check

Reply:
TBH, don't like this much logic in the Makefile as it prohibits `go test`

Reply:
Yeah, that's one of the reasons I was thinking that maybe we need to push that support into the e2e config parser

Reply:
modifying at run time in go, but github is not showing the changes...